app/models/pokemon.js

```diff
@@ -1,28 +1,39 @@
+"use strict";
+
 var mongoose = require('mongoose');
-var Schema   = mongoose.Schema;
+var Schema = mongoose.Schema;
 
+var pokemonGoBasicSchema = new Schema({
+    Number: {type: String},
+    Name: {type: String},
+    Classification: {type: String},
+    Types: [String],
+    Resistant: [String],
+    Weaknesses: [String],
+    Fast_Attacks: [{Name: {type: String}, Type: {type: String}, Damage: {type: Number}}],
+    Special_Attacks: [{Name: {type: String}, Type: {type: String}, Damage: {type: Number}}],
+    Weight: {
+        Minimum: {type: String},
+        Maximum: {type: String}
+    },
+    Height: {
+        Minimum: {type: String},
+        Maximum: {type: String}
+    },
+    FleeRate: Number,
+    Next_evolutions: [{Number: Number, Name: {type: String}}],
+    Previous_evolutions: [{Number: Number, Name: {type: String}}],
+    MaxCP: Number,
+    MaxHP: Number,
+    gender: {
+        abbreviation: {
+            type: String, enum: ['m', 'f', 'g', 'h'] //m=male, f=female, g=genderless, h=hermaphrodite
+        },
+        male_ratio: Number,
+        female_ratio: Number,
+        breedable: Boolean
+    }
 
-var pokemonSchema = new Schema({
-    id         : {type: String, required: true, unique: true},
-    created    : {type: Number},
-    downvotes  : {type: Number},
-    upvotes    : {type: Number},
-    latitude   : {type: Number},
-    longitude  : {type: Number},
-    pokemonId  : {type: String},
-    trainerName: {type: String},
-    userId     : {type: String},
-    deviceId   : {type: String},
-    category   : {type: [Boolean]},
-    gender     : {abbreviation: {type: String, enum: ['m', 'f', 'g', 'h']},
-                  male_ratio: Number,
-                  female_ratio: Number,
-                  breedable: Boolean},
-    weight     : {type: Number},
-    height     : {type: Number},
-    move       : {type: [Boolean]},
-    evolution  : {type: [String]},
-    pre_form   : {type: [String]}
 });
 
-module.exports = mongoose.model('pokemon', pokemonSchema);
\ No newline at end of file
+module.exports = mongoose.model('BasicPokemonDetail', pokemonGoBasicSchema);
```

Comment:
Oh no wait because then the collection would not be called basicpokemondetails as required for the api